src/test/java/org/opensearch/knn/TestUtils.java

```diff
@@ -32,6 +37,105 @@
     public static final String BWCSUITE_ROUND = "tests.rest.bwcsuite_round";
     public static final String TEST_CLUSTER_NAME = "tests.clustername";
 
+    //Generating index vectors using random function with a seed which makes these vectors standard and generate same vectors for each run.
+    public static float[][] randomlyGenerateStandardIndexVectors(int numVectors, int dimensions) {
```

Comment:
nit
```suggestion
    public static float[][] generateRandomStandardIndexVectors(int numVectors, int dimensions) {
```

Reply:
I felt the method name, "generateRandomStandard...." would create confusion, that's the reason I used "randomlyGenerateStandard...."